index.html

```diff
@@ -29,7 +29,7 @@ <h1 class="main-greeting"><b>Welcome!<br>
         </div>
     </main>
     <footer class="footer">
-        <p> © Copyright 2024 </p>
+        <p> © Copyright 2025 </p>
```

Comment:
The solution for #1 should not be to just change 2024 to 2025 but to use JavaScript so that it automatically displays current year in 2026 and so on.

Reply:
Fixed in 02d84fc, please verify.